### DISCUSSION TITLE
Public document creator email

### DISCUSSION THREAD

**Discussion by ChristianMingl3** [mxrch/GHunt#538] [Q&A] [answered]
Hi - is this tool able to identify the owner of a public google sheet? I have forgotten the email i used to sign up, and the only remnant of all my work is a single public document i made and sent to someone, and i was hoping to be able to get my email from that

**Comment by mxrch** [comment, marked as answer]
Hey, yes, you can use the subcommand `ghunt drive` for this

**Reply by ChristianMingl3** [reply]
okay, thank you!! :)
